book/src/dev/release-criteria.md

```diff
@@ -0,0 +1,113 @@
+# Go/No-Go Release Criteria
+
+## Alpha Release
+
+In the first alpha release, we want Zebra to participate in the network and replicate the Zcash chain state. We also want to validate proof of work and the transaction merkle tree, and serve blocks to peers.
+
+### System Requirements
+
+Our CI tests that Zebra builds, passes its tests, and syncs to tip on the following configurations:
+* Ubuntu Linux ...
+
+Our CI tests that Zebra builds and passes its tests on the following configurations:
+* recent macOS ...
+* recent Windows ...
+
+### Build Requirements
+
+Our CI builds Zebra with:
+* Rust stable
+* recent llvm
+* recent clang
+* recent libclang
+
+### Supported Platforms
+
+While Zebra is still in alpha, we don't guarantee support for any particular platforms.
+
+But you'll probably get the best results with a recent Ubuntu Linux, or the other platforms that our CI runs on.
+
+### Go/No-Go Criteria
```

Comment:
how are we going to track the color status on these items? My guess would be emojis. And are we going to do it in this file or out of band somehow?

:red_circle: :yellow_circle: :green_circle:

Reply:
Good questions, emojis are good. 
We should set aside some time (5-10 minutes) during the next Zebra Gardening calls (on 11/30 and 12/07) to go through the checklist and update the status (RAG) for each item by replacing the checkboxes with the appropriate emoji.